src/frontend/pyproject.toml

```diff
@@ -0,0 +1,118 @@
+[build-system]
+# needs a version that supports PEPs 517, 518, 621, 660
+# https://setuptools.pypa.io/en/latest/history.html#v61-0-0
+requires = ["setuptools>=61.0.0"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "CellpPofiler"
+version = "5.0.0.dev1"
+authors = [
```

Comment:
Several of these emails won't work anymore, do we want to remove? I notice Anne's also doesn't have one, so guessing we don't NEED to have them